airflow/ui/src/components/Select.tsx

```diff
@@ -0,0 +1,259 @@
+/*!
```

Comment:
Change this file name to `MultiSelect.tsx` to match the function name and differentiate from a standard `<select>`?